userguide/one_shot_object_detection/README.md

```diff
@@ -62,18 +61,60 @@ To learn more about deploying One-Shot to CoreML details, refer to the [Object D
 
 ## How it works
 
-In this section, we will go into detail on what happens under the hood of the `create` method. An OSOD model is trained in two steps: First, we use the starter images to generate synthetic data and annotations, and second, we train an object detector.
+In this section, we will go into detail on what happens under the hood of the `create` method. A One-Shot Object Detector model is trained in two steps: First, we use the starter images to generate synthetic data and annotations, and second, we train a model on the generated synthetic data.
```

Comment:
The second sentence that starts on this line is actually three sentences. I would re-work as follows:
```
A One-Shot Object Detector model is trained in two steps. First, the starter image(s) is used to generate synthetic data and annotations. Second, we train an object detector model on the generated synthetic data.
```